setup.py

```diff
@@ -291,61 +167,46 @@ def run(self):
                 # prevent crossfire with downstream scripts
                 del os.environ['CMAKE_ARGS']
                 log.info('Extra cmake args: {}'.format(extra_cmake_args))
+                cmake_args.extend(extra_cmake_args)
             cmake_args.append(TOP_DIR)
             subprocess.check_call(cmake_args)
 
-            if find_executable('make'):
-                build_args = [find_executable('make')]
-                # control the number of concurrent jobs
-                if self.jobs is not None:
-                    build_args.extend(['-j', str(self.jobs)])
-            else:
-                # Windows CI environment
-                build_args = [find_executable('cmake'), '--build', '.']
+            build_args = [CMAKE, '--build', os.curdir]
+            if self.jobs is not None and not WINDOWS:
+                build_args.extend(['--', '-j', str(self.jobs)])
             subprocess.check_call(build_args)
 
 
 class build_py(setuptools.command.build_py.build_py):
     def run(self):
         self.run_command('create_version')
-        self.run_command('build_proto')
         self.run_command('cmake_build')
+
+        for src in glob.glob(
+                os.path.join(CMAKE_BUILD_DIR, 'onnx', '*.py')):
+            dst = os.path.join(
+                TOP_DIR, os.path.relpath(src, CMAKE_BUILD_DIR))
+            self.copy_file(src, dst)
```

Comment:
is this part new?

Reply:
yep, it's needed to copy the cmake generated python files to setup.py's directory. This bug was not discovered because previously setup.py does its own protobuf autogen work (in addition to cmake).